melodic/distribution.yaml

```diff
@@ -527,6 +527,13 @@ repositories:
       type: git
       url: https://github.com/ros-perception/calibration.git
       version: hydro
+  camera_info_manager_py:
+    release:
+      tags:
+        release: release/melodic/{package}/{version}
+      url: https://github.com/appie-17/camera_info_manager_py-release.git
```

Comment:
In previous releases (like [Kinetic](https://github.com/ros/rosdistro/blob/master/kinetic/distribution.yaml#L1095)), this was released from the ros-perception organization (using the ros-gbp for the actual release).  Why the change?

Reply:
I changed the organization since it does not seem like it is actively maintained anymore for new releases... Since there are no issues running this package on newer releases, it made sense to release the package myself.

Reply:
We generally don't hand over maintainership for a package unless explicitly given permission by the original maintainer, or if all attempts to contact the original maintainer go unanswered for a period of time.  Can you document your attempts to contact the original maintainers of the package?